registry/Scripts/pre-reqs.ps1

```diff
@@ -68,6 +70,6 @@ $tp = Get-PfxCertificate -FilePath $pfxPath
 Write-Host "=> Certificate URL:" $kvSecret.Id
 Write-Host "=> Certificate thumbprint:" $tp.Thumbprint
 
-# Write-Host "Storing secret for sample user: admin"
-# $userSecret = ConvertTo-SecureString -String "admin" -AsPlainText -Force
-# Set-AzureKeyVaultSecret -VaultName $kvName -Name "admin" -SecretValue $userSecret | out-null
+Write-Host "Storing secret for sample user: $userName"
+$userSecret = ConvertTo-SecureString -String $userPass -AsPlainText -Force
+Set-AzureKeyVaultSecret -VaultName $kvName -Name $userName -SecretValue $userSecret -ContentType login | out-null
```

Comment:
I think `user credentials` may be slightly better than `login`

Reply:
Updated to user credential.